src/type/introspection.ts

```diff
@@ -323,6 +323,16 @@ export const __Type: GraphQLObjectType = new GraphQLObjectType({
         type: __Type,
         resolve: (type) => ('ofType' in type ? type.ofType : undefined),
       },
+      oneOf: {
+        type: GraphQLBoolean,
+        resolve: (type) => {
+          if (isInputObjectType(type)) {
+            return type.isOneOf;
+          }
+
+          return null;
```

Comment:
You don't need to return explicit null.
See the above fields, e.g. `enumValues`.

Reply:
Fixed :+1: